tests/airgapped/README.md

```diff
@@ -0,0 +1,91 @@
+# Testing Airgapped Installation
+
+For running the tests we expect an environment that can:
+1. Spin up LXC containers
+2. Have Docker, to pull images
+
+We need docker to pull all the images necessary and push them to the airgapped
+lxc container.
+
+## Setup
+
+We've prepared some scripts for setting up the environment
+```bash
+./tests/airgapped/setup/setup.sh
+```
+
+We've observed that in almost all cases we needed to reboot to be able to
+run docker as sudo and the lxc container to, initially, have network access.
```

Comment:
Compare with the docker multipass blueprint linked above. Iirc, it does not require a restart.

Reply:
The above comment refers to the need to be able to use lxc machine as well as docker from the same host machine, rather than running docker from inside the lxc machine.

So I'll resolve this for now and re-evaluate installing docker inside lxc with the profile as a dedicated discussion